src/bot/commands/report.ts

```diff
@@ -66,7 +65,7 @@ export default class ReportCommand extends Command {
       return interaction.reply(interactionInfo(`L'identifiant \`${jokeId}\` ne correspond à aucune blague connue.`));
     }
 
-    const raison = interaction.options.getString('raison', true);
+    const reason = interaction.options.getString('raison', true);
```

Comment:
ça te permettra d'éviter une réassignation du type plus bas (L150).

PS: Non testé, il y a moyen que le typage soit incompatible, si c'est le cas fais un `as Reason` à la fin.
```suggestion
    const reason: Reason = interaction.options.getString('raison', true);
```